nomad/structs/structs.go

```diff
@@ -2622,6 +2681,10 @@ type TaskGroup struct {
 	// Meta is used to associate arbitrary metadata with this
 	// task group. This is opaque to Nomad.
 	Meta map[string]string
+
+	// ReschedulePolicy is used to configure how the scheduler should
+	// retry failed allocations.
+	ReschedulePolicy *ReschedulePolicy
```

Comment:
Will have to add diffing code: https://github.com/hashicorp/nomad/blob/master/nomad/structs/diff.go